### DISCUSSION TITLE
WNS、WHS是“inf“的原因

### DISCUSSION THREAD

**Discussion by shengxsheng** [bol-edu/HLS-SOC-Discussions#116] [General]
想請問Synthesis完之後，WNS、WHS是“inf“的原因和解決辦法
![11](https://github.com/bol-edu/HLS-SOC-Discussions/assets/101039155/9c23cf65-a9c9-43e1-b7f6-443716187d86)

**Comment by ZheChen-Bill** [comment]
因為你沒有設定clock constraint。

**Reply by shengxsheng** [reply]
有依照workbook步驟，加入clock constraints
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/101039155/9b93ea25-5b70-40a3-a30b-2fa9100a7974)

**Comment by huankai0731** [comment]
同學您好，我也有遇到相同的問題，後來是參考了以下的檔案才解決。
供你參考看看，希望可以解決你的問題。
https://picture.iczhiku.com/resource/eetop/wykdDhqoFhsGhmNm.pdf
